splunk/resource_splunk_admin_saml_groups.go

```diff
@@ -70,8 +69,11 @@ func adminSAMLGroupsRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	// an empty entry (with no error) means the resource wasn't found
+	// mark it as such so it can be re-created
 	if entry == nil {
-		return fmt.Errorf("Unable to find resource: %v", name)
+		d.SetId("")
+		return nil
```

Comment:
A missing remote resource isn't always a failure/error scenario.  Often it just means the resource should be marked as destroyed so it can be removed from state and re-created.